WTWSMS/interface/WtWSMS_eventwindow.gfx

```diff
@@ -191,11 +196,16 @@ spriteTypes = {
 		texturefile = "gfx\\event_pictures\\WtWSMS\\warband_on_hill.tga"
 		allwaystransparent = yes
 	}
+	spriteType = {
+		name = "GFX_evt_warband_on_hill"
+		texturefile = "gfx\\event_pictures\\WtWSMS\\warband_on_hill.tga"
+		allwaystransparent = yes
+	}
```

Comment:
Why does this picture need two entries?

Reply:
It's used for different events, GFX_evt_battle_saxon is for battle events